src/server/handlers/launchpad.js

```diff
@@ -387,5 +383,6 @@ export const requestSnapBuilds = (req, res) => {
         }
       });
     })
-    .catch((error) => responseError(res, error));
+    .catch((error) => prepareError(error))
+    .catch((error) => res.status(error.status).send(error.body));
```

Comment:
As `prepareError` seems to be used mostly just to catch it again and return, maybe we could have `prepareErrorResponse(error, res)` instead that would format the error if necessary and return it as the response? This would avoid this a bit strange double `catch` pattern where we throw prepared error just to catch and return it in response.